build.gradle

```diff
@@ -33,9 +33,25 @@ buildscript {
 }
 
 apply plugin: 'com.palantir.git-version'
-apply plugin: 'com.palantir.baseline'
+
 apply plugin: 'com.palantir.consistent-versions'
 
+// expanded version of 'com.palantir.baseline' because errorprone doesn't work on Java12 yet
+apply plugin: 'com.palantir.baseline-config'
+apply plugin: 'com.palantir.baseline-circleci'
+allprojects {
+    apply plugin: 'com.palantir.baseline-checkstyle'
+    apply plugin: 'com.palantir.baseline-scalastyle'
+    apply plugin: 'com.palantir.baseline-eclipse'
+    apply plugin: 'com.palantir.baseline-idea'
+    // apply plugin: 'com.palantir.baseline-error-prone'
```

Comment:
Seems like it's a known issue that error-prone 2.3.3 doesn't work with Java 12: https://github.com/google/error-prone/issues/1106